GraphView/po/pl-local.po

```diff
@@ -0,0 +1,634 @@
+msgid "Dark (default)"
+msgstr "Ciemny (domyślny)"
+
+#: GraphView/avatars.py:616
+msgid "Light"
+msgstr "Jasny"
+
+#: GraphView/avatars.py:62
+msgid "Cartoon"
+msgstr "Rysunek"
+
+#: GraphView/graphview.gpr.py:34 GraphView/graphview.py:179
+msgid "Graph View"
+msgstr "Graph View"
+
+#: GraphView/graphview.gpr.py:35
+msgid "Charts"
+msgstr "Wykresy"
+
+#: GraphView/graphview.gpr.py:36
+msgid "Dynamic and interactive graph of relations"
+msgstr "Dynamiczny i interaktywny graf relacji"
+
+#: GraphView/graphview.gpr.py:49
+msgid ""
+"Graphview Warning:  Goocanvas 2 or 3 (https://wiki.gnome.org/action/show/"
+"Projects/GooCanvas) is required for this view to work"
+msgstr ""
+"Ostrzeżenie GraphView: Do działania wymagany jest Goocanvas 2 lub 3"
+"(https://wiki.gnome.org/action/show/"
+
+#: GraphView/graphview.gpr.py:54
+msgid ""
+"Graphview Warning:  GraphViz (http://www.graphviz.org) is required for this "
+"view to work"
+msgstr ""
+"Ostrzeżenie GraphView: Do działania wymagany jest GraphViz"
+"(http://www.graphviz.org)"
+
+#: GraphView/graphview.gpr.py:63
+msgid "Graphview Failed to Load"
+msgstr "GraphView nie został załadowany"
+
+#: GraphView/graphview.gpr.py:64
+#, python-format
+msgid ""
+"\n"
+"\n"
+"Graphview is missing python modules or programs.\n"
+"%smust be installed.\n"
+"\n"
+"For now, it may be possible to install the files manually. See\n"
+"<a href=\"https://gramps-project.org/wiki/index.php?title=Graph_View\" title="
+"\"https://gramps-project.org/wiki/index.php?title=Graph_View\">https://"
+"gramps-project.org/wiki/index.php?title=Graph_View</a> \n"
+"\n"
+"To dismiss all future Graphview warnings click Dismiss."
+msgstr ""
+"\n"
+"\n"
+"W GraphView brakuje modułów lub programów Pythona.\n"
+"%s musi być zainstalowane.\n"
+"\n"
+"Na razie możliwe jest ręczne zainstalowanie plików. Zobacz\n"
+"<a href=\"https://gramps-project.org/wiki/index.php?title=Graph_View\" title="
+"\"https://gramps-project.org/wiki/index.php?title=Graph_View\">https://"
+"gramps-project.org/wiki/index.php?title=Graph_View</a> \n"
+"\n"
+"Aby odrzucić wszystkie przyszłe ostrzeżenia GraphView, kliknij Odrzuć."
+
+#: GraphView/graphview.gpr.py:80
+msgid " Dismiss "
+msgstr " Odrzuć "
+
+#: GraphView/graphview.gpr.py:81
+msgid "Continue"
+msgstr "Kontynuuj"
+
+#: GraphView/graphview.py:681 GraphView/graphview.py:3567
+msgid "Show images"
+msgstr "Pokaż obrazki"
+
+#: GraphView/graphview.py:684
+msgid "Show avatars"
+msgstr "Pokaż awatary"
+
+#: GraphView/graphview.py:687 GraphView/graphview.py:3575
+msgid "Highlight the home person"
+msgstr "Wyróżnij osobę główną"
+
+#: GraphView/graphview.py:691 GraphView/graphview.py:3583
+msgid "Show full dates"
+msgstr "Pokaż pełne daty"
+
+#: GraphView/graphview.py:695 GraphView/graphview.py:3591
+msgid "Show places"
+msgstr "Pokaż miejsca"
+
+#: GraphView/graphview.py:698 GraphView/graphview.py:2877
+msgid "Default"
+msgstr "Domyślny"
+
+#: GraphView/graphview.py:704
+msgid "Place format"
+msgstr "Format miejsc"
```

Comment:
Question if it should be "Format miejsc" or "Format miejsca" perhaps could be "Format dla miejsc"? No strong opinion on that.

Reply:
You're right. Correct.